src/pages/test-report/components/uncouple-recouple/uncouple-recouple.html

```diff
@@ -0,0 +1,36 @@
+<ion-row>
+  <ion-col col-20 no-padding>
+    <competency-button class="uncouple-recouple-tick"
+                       [onTap]="toggleUncoupleRecouple"
```

Comment:
This formatting is not aligned with other html files. Just use 2 spaces please.

Reply:
done